sncrs/data/serializers.py

```diff
@@ -279,3 +280,14 @@ def create(self, validated_data):
                     ))
             quote.speakers.add(speaker_object.id)
         return quote
+
+
+class LessonSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Lesson
+        fields = [
+            'id',
+            'text',
+            'url',
+            ]
```

Comment:
No need to change it, but I just recently learned that if you want to, you can do this to include all fields more concisely!
```suggestion
        fields = "__all__"
```